pep-0696.rst

```diff
@@ -184,16 +187,93 @@ default to the type of ``start`` and step default to ``int | None``.
    StopT = TypeVar("StopT", default=StartT)
    StepT = TypeVar("StepT", default=int | None)
 
-   class slice(Generic[StartT, StopT, StepT]): ...  # Valid
+   class slice(Generic[StartT, StopT, StepT]): ...
+
+   reveal_type(slice())                        # type is slice[int, int, int | None]
+   reveal_type(slice[str]())                   # type is slice[str, str, int | None]
+   reveal_type(slice[str, bool, timedelta]())  # type is slice[str, bool, timedelta]
+
+When using a ``TypeVarLike`` as the default to another ``TypeVarLike``.
+Where ``T1`` is the default for ``T2`` the following rules apply.
+
+Scoping Rules
+~~~~~~~~~~~~~
+
+``T1`` must be used before ``T2`` in the parameter list of the generic
+or be bound in an outer class or function scope.
+
+.. code:: py
+
+   DefaultT = TypeVar("DefaultT", default=T)
 
-   reveal_type(slice())                       # type is slice[int, int, int | None]
-   reveal_type(slice[str]())                  # type is slice[str, str, int | None]
-   reveal_type(slice[str, str, timedelta]())  # type is slice[str, str, timedelta]
+   class Foo(Generic[T, DefaultT]): ...   # Valid
+   def bar(x: T, y: DefaultT): ...        # Valid
+   class Foo(Generic[T]):
+       class Bar(Generic[DefaultT]): ...  # Valid
+   def outer(x: T):
+       def inner(y: DefaultT): ...        # Valid
 
-   StartT = TypeVar("StartT", default="StopT")
+   StartT = TypeVar("StartT", default="StopT")  # Swapped defaults around from previous example
    StopT = TypeVar("StopT", default=int)
    class slice(Generic[StartT, StopT, StepT]): ...
-                       ^^^^^^  # Invalid: ordering does not allow StopT to bound yet
+                     # ^^^^^^ Invalid: ordering does not allow StopT to be bound
+   def baz(x: DefaultT, y: T): ...
+            # ^^^^^^^^ Invalid: ordering does not allow DefaultT to be bound
+
+Bound Rules
+~~~~~~~~~~~
+
+``T2``'s bound must be a subtype of ``T1``'s bound.
+
+.. code:: py
+
+   T = TypeVar("T", bound=float)
+   TypeVar("Ok", default=T, bound=int)        # Valid
+   TypeVar("AlsoOk", default=T, bound=float)  # Valid
+   TypeVar("Invalid", default=T, bound=str)   # Invalid: str is not a subtype of float
+
+Constraint Rules
+~~~~~~~~~~~~~~~~
+
+``T2``'s constraint must either contain the bound of ``T1`` or the
+constraints of ``T2`` must be a superset of the constraints of ``T1``.
+
+.. code:: py
+
+   T1 = TypeVar("T1", bound=int)
+   TypeVar("Ok", int, str, default=T1)                # Valid
+   TypeVar("Invalid", float, str, default=T1)         # Invalid: upper bound int is incompatible with constraints float or str
+
+   T1 = TypeVar("T1", int, str)
+   TypeVar("AlsoOk", int, str, bool, default=T1)      # Valid
+   TypeVar("AlsoInvalid", bool, complex, default=T1)  # Invalid: {bool, complex} is not a superset of {int, str}
+
+
+``TypeVarLike``\ s As Parameters To Generics
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Another typographical nit, but since this was specifically changed to use title case for h3+ headings (most PEPs I've seen tend to use sentence case for those headings, but up to you—just be consistent), you don't capitalize articles, short conjunctions, etc. in title case. Also, FYI, for this an the rest, you can just escape the `s` directly rather than the space before the `s`.

```suggestion
``TypeVarLike``\s as Parameters to Generics
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```